lib/domain.js

```diff
@@ -33,6 +33,9 @@ module.exports = function(logger) {
 
   return function domainMiddleware(req, res, next) {
     var requestDomain = domain.create();
+    //make sure req and res are associated to the newly created domain
```

Comment:
Minor: I'd include a link to [domain explicit binding](https://nodejs.org/api/domain.html#domain_explicit_binding) to describe "why" and not "what" the code does.